mate-academy/layout_landing-page
# Develop

❗❗❗ IMPORTANT ❗❗❗
I have some issues with pushing changes for gh-pages.
eg. button View all exhibitions does not have padding top and botton in css file created, whilst develop branch events.scss have it. repushed a few times with some minor changes to file, and redeployed - still the same issue.

I have to recreate form to push max changes that were not picked up with the prev forked repository. 
prev pull request: https://github.com/mate-academy/layout_landing-page/pull/1825 with mentor comments what to fix.

**Project.**

The Met website learning project.
css, html, no JS

Preview link wont work because of style file path in index html.

instructions on how to preview file locally:

- clone project to your pc
- run "npm i" within project folder
- switch to develop branch
- run "npm start" within project folder
- check site on http://localhost:1234/

[preview link](https://taniakolesnik.github.io/layout_landing-page/)

[figma project](https://www.figma.com/file/lSR1m42L9YwzQwzzxKwHpw/THE-MET)
